templates/fedora.tpl.yaml

```diff
@@ -6,10 +6,8 @@ metadata:
   annotations:
     openshift.io/display-name: "Fedora 23+ VM"
     description: >-
-      This template can be used to create a VM suitable for
-      Fedora 23 and newer.
-      The template assumes that a PVC is available which is providing the
-      necessary Fedora disk image.
+      Template for Fedora 23 VM or newer.
```

Comment:
Please change this to Fedora 31 VM or newer.
Please change `openshift.io/display-name` annotation as well

Reply:
Done